### ISSUE TITLE
Drop untill click in dropzone

### ISSUE THREAD

**Issue by gags88** [reppners/ngx-drag-drop#78]
Is there any way not to drop the item untill user make a click in dropzone.

Demonstrated in GIF attached...
![screen-capture](https://user-images.githubusercontent.com/9352091/67494564-fdebb900-f63e-11e9-8ed2-ece9be65dd75.gif)

**Comment by reppners** [comment]
I don't think this is easily possible because of how tightly drag and drop is coupled to the mousedown / mousemove / mouseup workflow. This module is just a wrapper around HTML drag and drop capabilities and as such also tied to it's limitations/implementations. 

That being said this would need some fiddling around with to get to a definite yes/no answer. It may be possible by holding state and based on that state stopPropagation/preventDefault of the corresponding mouse events.. but there be dragons 🤷‍♂️ 😅
